src/packages/v4/components/modals/V4TokenHoldersModal/DownloadTokenHoldersModal.tsx

```diff
@@ -23,27 +23,34 @@ export function DownloadTokenHoldersModal({
   onCancel: VoidFunction | undefined
 }) {
   const { projectId } = useJBContractContext()
+  const chainId = useJBChainId()
 
   const [blockNumber, setBlockNumber] = useState<number>()
   const [loading, setLoading] = useState<boolean>()
 
   // Use block number 5 blocks behind chain head to allow for subgraph being a bit behind on indexing.
   const { data: latestBlockNumber } = useBlockNumber({ behindChainHeight: 5 })
 
-  const { data } = useSubgraphQuery({
-    document: ParticipantsDownloadDocument,
+  const { data: project } = useProjectQuery({
+    client: bendystrawClient,
+    variables: {
+      projectId: Number(projectId),
+      chainId: Number(chainId),
+    },
+    skip: !projectId || !chainId,
+  })
+
+  const { data: _participants } = useParticipantsQuery({
+    client: bendystrawClient,
     variables: {
       where: {
-        projectId: Number(projectId),
+        suckerGroupId: project?.project?.suckerGroupId,
```

Comment:
this will be a pretty standard bendystraw pattern. starting with `projectId` + `chainId`, we query the matching project, and use `project.suckerGroupId` to get entities matching _all_ projects in that sucker group